lib/services/http-service.js

```diff
@@ -142,6 +144,17 @@ function httpServiceFactory(
             });
         }
 
+        //Swagger support
+        var swaggerConfig = {
+            appRoot: path.resolve(__dirname, '../../'),
+            configDir: path.resolve(__dirname, '../../swagger_config'),
+            swagger: path.resolve(__dirname, '../../static/monorail-2.0.yaml'),
```

Comment:
I think @jlongever has a PR that could leverage `api-doc` to generate `swagger.json`, which I think could just satisfy swagger UI and make it work. Why not align with his solution ? If in that way, do we still need the `monorail-2.0.yaml` ?